j2lint/cli.py

```diff
@@ -153,43 +155,68 @@ def run(args=None):
         settings.output = "json"
         logger.debug("JSON output enabled")
 
-    lint_issues = {}
+    lint_errors = {}
+    lint_warnings = {}
     files = get_files(file_or_dir_names)
 
     # Get linting issues
     for file_name in files:
         runner = Runner(collection, file_name, checked_files)
-        if file_name not in lint_issues:
-            lint_issues[file_name] = []
-        lint_issues[file_name].extend(runner.run())
+        if file_name not in lint_errors:
+            lint_errors[file_name] = []
+        if file_name not in lint_warnings:
+            lint_warnings[file_name] = []
+        j2_errors, j2_warnings = runner.run()
+        lint_errors[file_name].extend(j2_errors)
+        lint_warnings[file_name].extend(j2_warnings)
 
     # Remove temporary file
     if stdin_filename:
         os.unlink(stdin_filename)
 
     # Sort and print linting issues
-    found_issues = False
-    json_output = []
-    if lint_issues:
-        for key, issues in lint_issues.items():
-            if len(issues):
-                if not found_issues:
-                    found_issues = True
+    found_errors = False
+    found_warnings = False
+    total_errors = 0
+    total_warnings = 0
+    json_output = {}
+    if lint_errors:
+        for key, errors in lint_errors.items():
+            if len(errors):
+                total_errors = total_errors + len(errors)
+                if not found_errors:
+                    found_errors = True
```

Comment:
No need for found_errors variable, instead you can check on if total_errors?

Reply:
updated